packages/ethereum/contracts/HoprChannels.sol

```diff
@@ -465,24 +400,22 @@ contract HoprChannels is IERC777Recipient, ERC1820Implementer {
         require(counterparty != address(0), "counterparty must not be empty");
         require(nextCommitment != bytes32(0), "nextCommitment must not be empty");
         require(amount != uint256(0), "amount must not be empty");
-        (,,, Channel storage channel) = _getChannel(
+        (,,, Channel storage earningChannel) = _getChannel(
```

Comment:
I guess we are moving away from the concept of updating both channels? this means that we can also send users their HOPR as soon after they redeem ticket since we don't focus on gas costs atm

Reply:
actually that takes us away from the spec, so receiving tokens immediately might not be a good idea